common/internal_model/src/main/scala/uk/ac/wellcome/models/parse/FreeformDateParser.scala

```diff
@@ -1,61 +0,0 @@
-package uk.ac.wellcome.models.parse
-
-import uk.ac.wellcome.models.work.internal.InstantRange
-import fastparse._, NoWhitespace._
-import DateParserImplicits._
-
-/**
-  *  Attempts to parse freeform text into a date or date range
-  */
-object FreeformDateParser extends Parser[InstantRange] with DateParserUtils {
-
-  def parser[_: P] =
-    Start ~ inferredTimePeriod | timePeriod ~ End
-
-  override def apply(input: String): Option[InstantRange] =
-    super.apply(input) map (_ withLabel input)
-
-  def inferredTimePeriod[_: P] =
-    ("[".? ~ timePeriod ~ "]") map (_ withInferred true)
-
-  def timePeriod[_: P] =
-    dateRange |
-      year.toInstantRange |
-      calendarDate.toInstantRange |
-      monthAndYear.toInstantRange
-
-  def dateRange[_: P] =
-    (year to year).toInstantRange |
-      (calendarDate to calendarDate).toInstantRange |
-      (monthAndYear to monthAndYear).toInstantRange |
-      (month to monthAndYear).toInstantRange |
-      (day to calendarDate).toInstantRange
```

Comment:
Am I right in thinking you binned all these because they weren't used anywhere prior to this patch?

Reply:
Not quite - these were the old Sierra parsing rules which are a strict subset of the Calm rules, so binning them was basically just removing duplication. See comment above re the "inferred" stuff